### PR TITLE
traffic sans

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which includes the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

File: solutions/usaco-917.mdx [hunk 1]
```diff
@@ -11,12 +11,91 @@ author: Brad Ma
...
+#include <vector>
+#include <string>
+#include <algorithm>
+
+using std::cout;
+using std::endl;
+using std::vector;
+
+// An arbitrary number that's large compared to the ones in the problem.
+const int LARGE = 1e9;
```
Comment (ryanchou-dev) [inline solutions/usaco-917.mdx:34]: umm, maybe `LARGE` is too vague of a word, how bout `MAX_V` or smth

File: solutions/usaco-917.mdx [hunk 2]
```diff
@@ -11,12 +11,91 @@ author: Brad Ma
...
+		} else if (segment_type[m] == "on") {
+			low -= end[m];
+			high -= start[m];
+			// Set to zero if low becomes negative
+			low = std::max(0, low);
+		}
+	}
+
+	std::ofstream written("traffic.out");
+	written << low << ' ' << high << endl;
```
Comment (ryanchou-dev) [inline solutions/usaco-917.mdx:69]: (optional)

change to `write`

**Review by ryanchou-dev — APPROVED** [review]

File: solutions/usaco-917.mdx (hunk 1 above)
Comment (SansPapyrus683) [reply solutions/usaco-917.mdx:34]: it isn't, `MAX_V` is 1000

Comment (ryanchou-dev) [reply solutions/usaco-917.mdx:34]: then why don't you just set the constant to `1001`?

Comment (SansPapyrus683) [reply solutions/usaco-917.mdx:34]: i feel like that would be too small

Comment (ryanchou-dev) [reply solutions/usaco-917.mdx:34]: errr, ok

**Review by maggie-j-liu — APPROVED** [review]

**Pull request merged**